tests/snapshots/FeatureAPI.kt

```diff
@@ -69,40 +69,193 @@ class FeatureAPI : GBIORxWebServices() {
-    ) : Jsonable  {
+    ) : Jsonable {
     
-        open class Dev (
-            @SerializedName("id") val id: String? = null,  // A unique identifier for the device 
-            @SerializedName("os") val os: String? = null, 
-            @SerializedName("ver") val ver: String? = null 
-        ) : Jsonable 
+        override fun toString() = "ClientData(locale=$locale, ver=$ver, dev=$dev, app=$app, user=$user, ctx=$ctx)"
+        
+        override fun hashCode(): Int {
+            var result:Int = locale.hashCode()
+            result = 31 * result + ver.hashCode()       
+            result = 31 * result + dev.hashCode()       
+            result = 31 * result + app.hashCode()       
+            result = 31 * result + user.hashCode()       
+            result = 31 * result + ctx.hashCode()       
+            return result
+        }
     
-        open class App (
-            @SerializedName("id") val id: String? = null, 
-            @SerializedName("ver") val ver: String? = null, 
-            @SerializedName("hr") val hr: Double? = null  // Hours since app install 
-        ) : Jsonable 
+        override fun equals(other: Any?): Boolean {
+            if (this === other) return true
+            if (javaClass != other?.javaClass) return false
     
-        open class User (
-            @SerializedName("country") val country: String? = null,  // The country in which the user is acting or to which they are assigned 
-            @SerializedName("anon_id") val anonId: String? = null  // If the user is not an authenticated user, an anonymous identifier for them (such as a guid) 
-        ) : Jsonable 
+            other as ClientData
+    
+            if (locale != other.locale) return false
+            if (ver != other.ver) return false
+            if (dev != other.dev) return false
+            if (app != other.app) return false
+            if (user != other.user) return false
+            if (ctx != other.ctx) return false
+    
+            return true
+        }
+    
+            open class Dev (
+                @SerializedName("id") val id: String? = null,  // A unique identifier for the device 
+                @SerializedName("os") val os: String? = null, 
+                @SerializedName("ver") val ver: String? = null 
+            ) : Jsonable {
+            
+                override fun toString() = "Dev(id=$id, os=$os, ver=$ver)"
+                
+                override fun hashCode(): Int {
+                    var result:Int = id.hashCode()
+                    result = 31 * result + os.hashCode()       
+                    result = 31 * result + ver.hashCode()       
+                    return result
+                }
+            
+                override fun equals(other: Any?): Boolean {
+                    if (this === other) return true
+                    if (javaClass != other?.javaClass) return false
```

Comment:
So this breaks content equality for subclasses that have the same content as a super class.  I'm fine with that, just double checking intentions.

Reply:
Yeah I'm ok with that.